package.json

```
@@ -46,7 +46,7 @@
     "typecheck": "nx run-many --target=typecheck --all --parallel"
   },
   "engines": {
-    "node": "^12.22.0 || ^14.17.0 || >=16.0.0"
+    "node": "^14.17.0 || >=16.0.0"
```

Using minimum Node 14.18 would enable full support for `node:` protocol imports: https://2ality.com/2021/12/node-protocol-imports.html

```suggestion
    "node": "^14.18.0 || >=16.0.0"
```

---

I'd also recommend dropping support for Node 17 which is end-of-life already:

```suggestion
    "node": "^14.18.0 || ^16.0.0 || >=18.0.0"
```

---

sounds good to me - I updated #5917 (which was left open).

---

Not sure if we want to update other values for now as I think it would be best to keep these in line with next major ESLint version 🤔

---

ESlint will most likely use the same Node engine requirement that I recommended in its next major version.